calculate.py

```diff
@@ -64,17 +64,22 @@ def get_10year(lang):
 
 
 def get_bitfinex():
-    url = "https://api-pub.bitfinex.com/v2/ticker/tBTCUSD"
-    res = requests.get(url)
-    rates = res.json()
-    last_price = rates[6]
-    sathkd = round((1/last_price)*100000000/7.75)  # assume exch rate is 7.75
+    satDenominator = 100000000
+    btcDataURL = "https://api-pub.bitfinex.com/v2/ticker/tBTCUSD"
+    btcRates = requests.get(btcDataURL).json()
+    btcLastPrice = btcRates[6]
+    
+    hkdDataURL = "https://v6.exchangerate-api.com/v6/22d53ba1468c061792921328/latest/HKD"
```

Comment:
This URL does not work, if its not a free API, we should not use, as page can be reloaded hundreds of times per day based on the way the server is setup

 hkdDataURL = "https://v6.exchangerate-api.com/v6/22d53ba1468c061792921328/latest/HKD"

Reply:
It is better to use a static once-a-day Forex exchange rate instead of trying to get refresh rate every second here.

Reply:
oh... sorry. It works perfectly now

Reply:
The free API is expected for frequent use as HKD exchange band 7.75 - 7.85 is not guaranteed